### DISCUSSION TITLE
help to frigate on homeassistant with 2nverso camera

### DISCUSSION THREAD

**Discussion by sawenz** [blakeblackshear/frigate#6743] [General]
i am running out of ideas.
I can see the stream of the camera under cameras.
the problem is: under events both thumbnails and clips are displayed. When i go to clips it shows: 
"The media could not be loaded, either because the server or network failed or because the format is not supported."
the video is downloadable and can be viewed with vlc viewer.
What am I doing wrong?

This is my config below attached

**Comment by sawenz** [comment]
[config.txt](https://github.com/blakeblackshear/frigate/files/11689486/config.txt)

**Reply by NickM-27** [reply]
this doesn't seem right, if you are using an mjpeg camera then you need to transcode it to h264 as the docs explain https://docs.frigate.video/configuration/camera_specific#mjpeg-cameras

**Reply by sawenz** [reply]
thanks

**Comment by sawenz** [comment]
<img width="542" alt="Screenshot 2023-06-08 161108" src="https://github.com/blakeblackshear/frigate/assets/103325771/b4b11704-7cfa-455b-be47-f1af3883f9dd">

**Comment by sawenz** [comment]
<img width="796" alt="Screenshot 2023-06-08 161053" src="https://github.com/blakeblackshear/frigate/assets/103325771/13e81070-0534-4a42-8ad2-bc292f9c3c5b">
